### DISCUSSION TITLE
I cross compiled obtr, but couldn't receive the message for the light bulb correctly.

### DISCUSSION THREAD

**Discussion by xjshiMax** [openthread/openthread#9893] [Q&A]
My testing method is to cross compile the openwrt version of OTBR on Ubuntu, then port it to the device and run it normally. The nanoleaf app thread can search for it, but it is grayed out. My OTBR also cannot receive the SRP message of the light board. I checked some information and said it may be a problem with the routing table, but I don't know how to configure it.

debug
$ version
OPENTHREAD/thread-reference-20230706-380-gb9dcdbca4; POSIX; Feb 27 2020 21:05:12
$ state
leader
$ rloc16
7000
$ extaddr
3ece76e319802832
$ ipaddrs
Error 35: InvalidCommand
$ ipmaddrs
Error 35: InvalidCommand
$ channel
15
$ panid
0x8316
$ extpanid
4c055463ec612758
$ netdata show
Prefixes:
fda7:4b16:7a51:1::/64 paos low 7000
Routes:
fda7:4b16:7a51:2:0:0::/96 sn low 7000
fc00::/7 sa med 7000
Services:
44970 01 53000500000e10 s 7000
44970 5d fd81121e9c3c844001109b8b8437390cd11f s 7000
Contexts:
fda7:4b16:7a51:1::/64 1 c
Commissioning:
36323 7000 - 00
$ netdata show -x
0b0e8001010d09700053000500000e1003140040fda74b167a51000105047000f1000702114003130060fda74b167a5100020000000001037000e003080007fc01037000100b1981015d0d147000fd81121e9c3c844001109b8b8437390cd11f080b090270000b028de3080100
$ partitionid
183978791
$ leaderdata
Partition ID: 183978791
Weighting: 64
Data Version: 23
Stable Data Version: 159
Leader Router ID: 28
$ child table
| ID  | RLOC16 | Timeout    | Age        | LQ In | C_VN |R|D|N|Ver|CSL|QMsgCnt|Suprvsn| Extended MAC     |
+-----+--------+------------+------------+-------+------+-+-+-+---+---+-------+-------+------------------+

$ childip
$ neighbor table
| Role | RLOC16 | Age | Avg RSSI | Last RSSI |R|D|N| Extended MAC     | Version |
+------+--------+-----+----------+-----------+-+-+-+------------------+---------+

$ router table
| ID | RLOC16 | Next Hop | Path Cost | LQ In | LQ Out | Age | Extended MAC     | Link |
+----+--------+----------+-----------+-------+--------+-----+------------------+------+
| 28 | 0x7000 |       63 |         0 |     0 |      0 |   0 | 3ece76e319802832 |    0 |

$ nexthop
| ID   |NxtHop| Cost |
+------+------+------+
|   28 |   28 |    0 |
$ eidcache
$ srp client state
Enabled
$ srp client host
name:(null), state:Removed, addrs:[]
$ srp client service
$ srp client server
[fd81:121e:9c3c:8440:110:9b8b:8437:390c]:53535
$ srp server state
running
$ srp server addrmode
unicast
$ srp server host
$ srp server service
$ br state
running
$ br omrprefix
Local: fda7:4b16:7a51:1::/64
Favored: fda7:4b16:7a51:1::/64 prf:low
$ br onlinkprefix
Local: fd4c:554:63ec:2758::/64
Favored: fd4c:554:63ec:2758::/64
$ br prefixtable
prefix:fdae:e2ab:d1b5:4ca6::/64, on-link:yes, ms-since-rx:50483, lifetime:1477, preferred:0, router:fe80:0:0:0:32ae:7bff:fee8:2faa (M:0 O:0 Stub:1)
$ br nat64prefix
Local: fda7:4b16:7a51:2:0:0::/96
Favored: fda7:4b16:7a51:2:0:0::/96 prf:low
$ bufferinfo
total: 256
free: 250
max-used: 29
6lo send: 0 0 0
6lo reas: 0 0 0
ip6: 0 0 0
mpl: 0 0 0
mle: 0 0 0
coap: 3 6 191
coap secure: 0 0 0
application coap: 0 0 0
Done

**Comment by jwhui** [comment]
The output of `neighbor table` is empty, which indicates that there are no other devices attached to your OTBR.

I would first validate that Active Operational Dataset matches exactly what your other Thread devices are configured with.
